src/libslic3r/PerimeterGenerator.cpp

```diff
@@ -1784,8 +1784,12 @@ void PerimeterGenerator::process_arachne()
             if (const auto& min_feature_size_opt = object_config->min_feature_size)
                 input_params.min_feature_size = min_feature_size_opt.value * 0.01 * min_nozzle_diameter;
 
-            if (const auto& min_bead_width_opt = object_config->min_bead_width)
-                input_params.min_bead_width = min_bead_width_opt.value * 0.01 * min_nozzle_diameter;
+            if (const auto &min_bead_width_opt = object_config->min_bead_width) {
```

Comment:
Choose min_bead_width or initial_layer_min_bead_width here would make the code cleaner.
Please ref to my other comment